src/Akka.Persistence.MongoDb/Journal/MongoDbJournal.cs

```diff
@@ -81,48 +94,68 @@ protected override void PreStart()
 
                 if (_settings.AutoInitialize)
                 {
-                    var modelForEntryAndSequenceNr = new CreateIndexModel<JournalEntry>(Builders<JournalEntry>
-                        .IndexKeys
-                        .Ascending(entry => entry.PersistenceId)
-                        .Descending(entry => entry.SequenceNr));
+                    var (perCallCts, unitedCts) = CreatePerCalLCts();
```

Comment:
Helper method to create a per-call CTS that is also linked to the global "operation CTS" is created here. We wrap both CTS in a `using` statement so they're safely disposed of afterwards.